### PR TITLE
Make datasources for Grafana dashboard configurable

### PR DESCRIPTION
**Pull request by Ornias1993**
Does what is states on the tin.
Adds a little selectionbox up-top to select the datasource instead of hardcoding one.

### CODE REVIEW

**Comment by mgdigital** [comment]
Thanks, looks good and I'll test it when I get a chance. Just a couple of small things:

- could you revert the casing change to `bitmagnet`?
- should the next version be `2` rather than `3`?
- what is that `"id": 79` referring to?

**Comment by Ornias1993** [comment]
> Thanks, looks good and I'll test it when I get a chance. Just a couple of small things:
> 
> * could you revert the casing change to `bitmagnet`?
> * should the next version be `2` rather than `3`?
> * what is that `"id": 79` referring to?

- Shouldn't names be cased like names?
- No idea that's what grafana came up with.

File: observability/grafana-dashboards/bitmagnet.json [hunk 1]
```diff
@@ -1395,19 +1410,57 @@
...
+    ]
   },
   "time": {
     "from": "now-6h",
     "to": "now"
   },
   "timepicker": {},
   "timezone": "",
-  "title": "bitmagnet",
+  "title": "Bitmagnet",
```
Comment (Ornias1993) [inline observability/grafana-dashboards/bitmagnet.json:1462]:
```suggestion
  "title": "bitmagnet",
```

File: observability/grafana-dashboards/bitmagnet.json [hunk 2]
```diff
@@ -1395,19 +1410,57 @@
...
   "uid": "c0abadd0-6633-40c4-a2fc-01408a8dfc4e",
-  "version": 1,
+  "version": 3,
```
Comment (Ornias1993) [inline observability/grafana-dashboards/bitmagnet.json:1464]:
```suggestion
  "version": 2,
```

**Comment by DyonR** [comment]
> * Shouldn't names be cased like names?
  
It just a style choice to have `bitmagnet` all lowercase. For example, the brand _adidas_ is also stylized with all lowercase.  
You would also write BitTorrent and not Bittorrent, and you also wouldn't write QBittorrent, but qBittorrent. It are all style choices of the authors.

**Comment by Ornias1993** [comment]
> > * Shouldn't names be cased like names?
> 
> It just a style choice to have `bitmagnet` all lowercase. For example, the brand _adidas_ is also stylized with all lowercase. You would also write BitTorrent and not Bittorrent, and you also wouldn't write QBittorrent, but qBittorrent. It are all style choices of the authors. Edit: One more example (to give a Dutch example); bol, formerly known as bol.com, is also a brand which uses only lowercase letters. (except in the history section and if sentences start with bol) You can see `bol` only written in lowercase: https://pers.bol.com/nl/het-verhaal/

You don't need to explain stylisation to me.
It's done frequently, Even I do it.

But for references it's still incorrect.

**Pull request merged**